### ISSUE TITLE
Sorry! Something went wrong.

### ISSUE THREAD

**Issue by dingzhixue** [kubeapps/kubeapps#2568]
### Steps to reproduce the issue:

1. [First Step]
helm install --version=3.9.2 kubeapps --namespace kubeapps bitnami/kubeapps --set useHelm3=true
2. [Second Step]

3. [and so on...]

### Version of Helm, Kubeapps and Kubernetes:

- Output of `helm version`:

```
(paste your output here)
```

- Output of `helm list -n <kubeapps-namespace> <kubeapps-release-name>`:

```
(paste your output here)
```

- Output of `kubectl version`:

```
(paste your output here)
```

**Comment by antgamdia** [comment]
It seems that something is happening with the old UI, it seems the ACE text editor is not able to be rendered.
Whereas I would recommend you to stick close to the latest Kubeapps versions, you still can workaround this error by:

`helm upgrade --version=3.9.2 kubeapps --namespace kubeapps bitnami/kubeapps --set useHelm3=true --set featureFlags.ui=clarity`

**Comment by andresmgot** [comment]
Indeed, we only support the latest major version of Kubeapps. Please use a more recent version, at the moment, the latest one available is 5.3.4:

```
helm install --version=5.3.4 kubeapps --namespace kubeapps bitnami/kubeapps
```

Please, reopen it if the problem persists after upgrading. If that's the case, please provide more information like which page is giving you the error or if you can see any more information in the browser console.
